src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/view-cache/view-cache.component.html

```diff
@@ -0,0 +1,14 @@
+<alert type="info"
+       *ngIf="status === vcs.ValueNone">
+  We're still trying to get your data, please wait.
+</alert>
+
+<alert type="warning"
+       *ngIf="status === vcs.ValueStale">
+  This information might be stale.
```

Comment:
And what is the consequence of this? Should I try again? What does "stale" mean? "Outdated"? Please provide some context.

Reply:
No need to do anything. 
This just informs that the data received is from cache and it might be a few seconds old.

Reply:
Then how about stating it as such? "Displaying previously cached data", maybe? I am concerned that "This information might be stale." confuses/irritates the user. I would expect some hint on how "stale" it might be and what could be done to refresh it.